nifi-nar-bundles/nifi-mongodb-bundle/nifi-mongodb-services/src/main/java/org/apache/nifi/mongodb/MongoDBLookupService.java

```diff
@@ -83,6 +84,7 @@
         .displayName("Projection")
         .description("Specifies a projection for limiting which fields will be returned.")
         .required(false)
```

Comment:
Is this change from another PR?

Reply:
No, I'm lazy and didn't want to do a full review for a few lines of changes to fix the mongo test suite.

Reply:
OK,  Can you explain what you are changing and why then?

Reply:
It has no validator, ergo it is invalid according to the test framework.